.travis.yml

```diff
@@ -45,13 +46,13 @@ install:
 script:
   - cargo generate-lockfile
   - if [[ $TRAVIS_OS_NAME = "linux" ]]; then
-      sudo apt-get remove -y qemu-user-static &&
-      sudo apt-get install -y qemu-user-static &&
       sh ci/run-docker.sh $TARGET;
     else
       cargo test --target $TARGET &&
       cargo test --target $TARGET --release;
     fi
+  # Travis can't cache files that are not readable by "others"
+  - chmod -R a+r $HOME/.cargo
```

Comment:
Yes, in case we ever add a dependency that brings a file that doesn't have the o+r bit set. FWIW, I originally found this problem with the bitflags crate; its .gitignore was `rwx------` iirc.